docs/user_guide/configuring.rst

```diff
@@ -87,6 +99,18 @@ You should also enable the edit option in your 'html_theme_options':
        "use_edit_page_button": True,
    }
 
+if you have a self-hosted Bitbucket instance, you can
```

Comment:
Capitalize

Reply:
Also perhaps move below the GitHub configuration, and add an appropriate header level for:

```rst
Self-Hosted Edit URL
--------------------

The default edit URLs will resolve to the main public sites such as
`https://github.com`, `https://bitbucket.com` and `https://gitlab.com`. 
These can be overridden in `html_context`:

.. code:: python

   html_context = {
       "github_url": "<your-github-url>",
       # "gitlab_url": "<your-gitlab-url>",
       # "bitbucket_url": "<your-bitbucket-url>",
   }
````